### PR TITLE
Select: fix input height of mini size

### PR DESCRIPTION
**Pull request by wubaiqing**
Error in nextTick: "TypeError: Cannot read property 'clientHeight' of undefined"

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
请附上这个 bug 的重现链接。

**Comment by wubaiqing** [comment]
@Leopoldthecoder 

JSFiddle：[https://jsfiddle.net/s1d78r9h/2/](https://jsfiddle.net/s1d78r9h/2/)

File: packages/select/src/select.vue [hunk 1]
```diff
@@ -495,7 +495,10 @@
           if (!this.$refs.reference) return;
           let inputChildNodes = this.$refs.reference.$el.childNodes;
           let input = [].filter.call(inputChildNodes, item => item.tagName === 'INPUT')[0];
-          input.style.height = Math.max(this.$refs.tags.clientHeight + 6, sizeMap[this.size] || 36) + 'px';
+          const tags = this.$refs.tags;
+          input.style.height = this.selected.length === 0
+            ? sizeMap[this.selectSize] + 'px'
```
Comment (Leopoldthecoder) [inline packages/select/src/select.vue]: 1.x 中没有 `selectSize`，请修改为 `size`

**Comment by wubaiqing** [comment]
@Leopoldthecoder 

Thank you for correcting my mistake.

**Comment by kimyLee** [comment]
when can I download the new version in cnpm, I also met this issue now, and @1.4.9  still keep this mistake now

**Pull request merged**